### PR TITLE
create simple todo list app

### PR DESCRIPTION
**Pull request by SimonLab**
ref: #16

wireframes
todolist-app

widgets tree
widgets-tree

widgets tree with inspector:
widget tree

### CODE REVIEW

File: .flutter-plugins-dependencies [hunk 1]
```diff
@@ -1 +1 @@
...
...me":"path_provider_windows","path":"C:\\\\Users\\\\simon\\\\flutter\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\path_provider_windows-0.0.4+3\\\\","dependencies":[]},{"name":"shared_preferences_windows","path":"C:\\\\Users\\\\simon\\\\flutter\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\shared_preferences_windows-0.0.1+3\\\\","dependencies":["path_provider_windows"]}],"web":[{"name":"shared_preferences_web","path":"C:\\\\Users\\\\simon\\\\flutter\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\shared_preferences_web-0.1.2+7\\\\","dependencies":[]}]},"dependencyGraph":[{"name":"path_provider_linux","dependencies":[]},{"name":"path_provider_windows","dependencies":[]},{"name":"shared_preferences","dependencies":["shared_preferences_linux","shared_preferences_macos","shared_preferences_web","shared_preferences_windows"]},{"name":"shared_preferences_linux","dependencies":["path_provider_linux"]},{"name":"shared_preferences_macos","dependencies":[]},{"name":"shared_preferences_web","dependencies":[]},{"name":"shared_preferences_windows","dependencies":["path_provider_windows"]}],"date_created":"2020-12-09 11:34:52.761290","version":"1.24.0-10.2.pre"}
```
Comment (nelsonic) [inline .flutter-plugins-dependencies]: > "This is a generated file; do not edit or check into version control."

I think for learning purposes, we can keep the file in Git. 💭

File: .flutter-plugins [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # This is a generated file; do not edit or check into version control.
+integration_test=C:\\Users\\simon\\flutter\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\integration_test-1.0.1\\
```
Comment (nelsonic) [inline .flutter-plugins]: windows? 😮 😉

Comment (SimonLab) [reply .flutter-plugins]: 😄 yes I know!! my laptop has some issues so I'm using the desktop at the moment (I tried to create a dual boot with ubuntu but some problems occurs, using wsl if necessary necessary)

File: .flutter-plugins-dependencies (hunk 1 above)
Comment (nelsonic) [reply .flutter-plugins-dependencies]: Up to you though. if you prefer to delete go for it. 👍

**Comment by SimonLab** [comment]
Application preview:


This PR is getting big now and I think it might be worth making it public on master to collect feedback.
The readme is not perfect but will be enhanced with future PR especially when other Flutter concepts are used on the application (e.g https://github.com/dwyl/flutter-todo-list-tutorial/pull/26)

**Comment by nelsonic** [comment]
@SimonLab do you think you might have some time for a Zoom call to walk us through this e.g. Friday afternoon? 💭

**Comment by SimonLab** [comment]
@nelsonic Friday afternoon sounds good for a demo 👍

**Review by nelsonic — APPROVED** [review]
@SimonLab thanks for the zoom-walkthrough of the App/UI/Code; looks great. 🎉

**Pull request merged**